README.md

```diff
@@ -63,7 +63,7 @@ The topics will be inferred by the browser. The browser will leverage a classifi
 * Topics can also be retrieved via request headers, and marked as observed and eligible for topics calculation via response headers.
     * This is likely to be considerably more performant than using the JavaScript API.
     * The request header can be sent along with fetch requests via specifying an option: `fetch(<url>, {browsingTopics: true})`.
-    * The request header will be sent on document requests when the list of topics is non-empty and the request is allowable (e.g., permission policy allows it, the context is secure, etc.).
+    * The request header will be sent on document requests when the request is allowable (e.g., permission policy allows it, the context is secure, etc.).
```

Comment:
This specifically mentions document requests, which I think we're less clear on at the moment.